src/common/Version.hpp

```diff
@@ -26,6 +26,7 @@ class Version
 
     const QString &version() const;
     const QString &commitHash() const;
+    const bool &isModified() const;
```

Comment:
Move the comment from `Version.cpp` to here instead, that way it's accessible whenever `isModified` is autocompleted